### DISCUSSION TITLE
Regional Icon

### DISCUSSION THREAD

**Discussion by eleifer-twilio** [twilio-labs/paste#1007] [Q&A] [answered]
Hi, 

The Regional team has the world icon throughout components in the console to show that an element/product is "Regional" and to give more context about Regional if clicked on. In most places it can be hovered on and clicked to see more info. We have reviewed this with the DSYS in the "Explore Product - Regional" discussion. We think of this as an explicit "Info" icon which serves for education and marketing. Customers understood the interaction when tested. 

We question if the icon needs to have text with it. For both clarity and accessibility. Our main concern is that this interaction is consistent and allows the user to learn about regional products. Please add your thoughts on this to the discussion and please let me know if you have any questions. Thank you!

Cheers, 
Emma

![Routing Modual Breakdown ](https://user-images.githubusercontent.com/73968607/101424268-a1ce0d80-38af-11eb-967f-531b89789cb6.png)

**Comment by eleifer-twilio** [comment]
@serifluous

**Comment by TheSisb** [comment, marked as answer]
> We question if the icon needs to have text with it. 

If this isn't information that a screen-reader user would need to have, you can make the icon decorative: https://paste.twilio.design/icons/usage-guidelines#making-an-icon-decorative-or-informative

If this is the only hint to tell the user this is "Regional" content, consider giving the icon a descriptive title or changing some copy around the UI.

> Our main concern is that this interaction is consistent and allows the user to learn about regional products. 

Are you concerned that this interaction may not be consistent? I don't think we have an opinion from the design system point of view as this is very product specific. Our recommendation is to test with your users and do what is best for your product experience.
